src/common/kafka/EV44Serializer.cpp

```diff
@@ -16,7 +16,8 @@
 //#undef TRC_LEVEL
 //#define TRC_LEVEL TRC_L_DEB
 
-static constexpr size_t TimeSize = sizeof(uint32_t);
+static constexpr size_t ReferenceTimeSize = sizeof(uint64_t);
+static constexpr size_t OffsetTimeSize = sizeof(uint32_t);
 static constexpr size_t PixelSize = sizeof(uint32_t);
```

Comment:
Just for good measure it might make sense to add a comment here. Especially the OffsetTime  is 
(later on) a little confusing since we then add OffsetTimeOffset to the stew :-)